test/cpp/interop/xds_interop_client.cc

```diff
@@ -154,11 +156,16 @@ class TestClient {
             std::chrono::system_clock::now() + std::chrono::seconds(INT_MAX);
       }
     }
+    SimpleRequest request;
+    request.set_response_size(absl::GetFlag(FLAGS_response_payload_size));
+    std::string payload(absl::GetFlag(FLAGS_request_payload_size), '\0');
```

Comment:
What's the contents of the payload here? Random memory contents? repeated `\0`s?

Reply:
Repeated `\0` for `request_payload_size` times.

Reply:
Done the same way here in the regular `interop_client.cc` https://github.com/grpc/grpc/blob/master/test/cpp/interop/interop_client.cc#L250

Reply:
I'm not to good with c, so I don't really know what this would do

Reply:
@yashykt Does this look OK?

Reply:
> I'm not to good with c, so I don't really know what this would do

And just for reference, https://cplusplus.com/reference/string/string/string/ this is the 6th version of the constructor.

```
fill (6) | string (size_t n, char c);

(6) fill constructor
Fills the string with n consecutive copies of character c.
```

Reply:
Thanks!